client/pages/sections/legacy-establishments.js

```diff
@@ -0,0 +1,29 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import RepeaterReview from './repeater/review';
+
+const LegacyEstablishments = ({ establishment, ...props }) => {
+
+  const {
+    name,
+    licenceNumber
+  } = establishment;
+
+  return (
+    <Fragment>
+      <h2>Primary establishment</h2>
+      <dl className="inline">
+      <dt>Establishment name: </dt>
+      <dd>{ name }</dd>
+      <dt>Licence number: </dt>
+      <dd>{ licenceNumber }</dd>
+      </dl>
+      <br/>
```

Comment:
This feels like it shouldn't be necessary between block elements. If you want to add whitespace then you should use CSS.